package.json

```diff
@@ -5,6 +5,10 @@
   "dependencies": {
     "@emotion/react": "^11.9.0",
     "@emotion/styled": "^11.8.1",
+    "@fortawesome/fontawesome-svg-core": "^6.1.1",
+    "@fortawesome/free-regular-svg-icons": "^6.1.1",
+    "@fortawesome/free-solid-svg-icons": "^6.1.1",
+    "@fortawesome/react-fontawesome": "^0.1.18",
```

Comment:
Remove all 4 of these. `React-icons` has you covered

Reply:
Literally remove every icon library you have besides `React-icons`. Feather be gone